src/components/SessionWrapper.js

```diff
@@ -33,7 +33,11 @@ const SessionWrapper = (Page) => {
     const language =
       req && req.body && req.body.language ? req.body.language : "en";
 
-    const currentPageTitle = PageTitles.getUrlPageTitle(req.url);
+    const currentPageTitle = PageTitles.getUrlPageTitle(
```

Comment:
Should probably move this down to the end and use validatorErrorsCleaned and allValidationErrorsCleaned